packages/flutter/lib/src/widgets/raw_menu_anchor.dart

```diff
@@ -590,6 +590,21 @@ class _RawMenuAnchorState extends State<RawMenuAnchor> with _RawMenuAnchorBaseMi
 
   @override
   Widget buildAnchor(BuildContext context) {
+    final Widget? overlayPortal =
```

Comment:
Why this change?

Reply:
We want the menu button and the menu have a parent-children relationship, but for `OverlayPortal`, its `child` and `overlayChildBuilder` are siblings relationship. To construct parent-children relationship, we use `RawMenuAnchor.builder` as the parent to wrap `OverlayPortal` which is constructed by `RawMenuAnchor.child` and `RawMenuAnchor.overlayBuilder`.

The disadvantage for this change is, if either `widget.child` or `widget.builder` is null, we are not able to have the parent-children relationship, but `SubmenuButton` is implemented by using both, so it should be okay.